ports/freedink/FreeDink.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+get_controls
+
+GAMEDIR=/$directory/ports/freedink
+CONFDIR="$GAMEDIR/conf/"
+BINARY=minife
```

Comment:
Shouldn't we use "minife.${DEVICE_ARCH}" ? or it is no more a requirement ?

Reply:
Requirement if a port is multiarch. I do it if it has potential for other arch.  This one I would not use for x86 for example.